docs/src/app_dev/app_val_tokenomics/index.rst

```diff
@@ -0,0 +1,121 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _app_val_tokenomics:
+
+Tokenomics Overview
+===================
+
+Following is an overview of the application and validator tokenomics for
+application developers or validator operators. It builds on the "`Canton
+Coin: A Canton-Network-native payment
+application <https://www.digitalasset.com/hubfs/Canton%20Network%20Files/Documents%20(whitepapers%2c%20etc...)/Canton%20Coin_%20A%20Canton-Network-native%20payment%20application.pdf?__hstc=169870847.16854726061d8b28be85af48c17588c4.1750870506327.1750870506327.1750870506327.1&__hssc=169870847.1.1750870506328&__hsfp=1243925796&_gl=1*1fkzmuj*_gcl_au*MTkyOTE1NjAyNC4xNzUwODcwNTA2*_ga*NDU1NzM2NzgyLjE3NTA4NzA1MDY.*_ga_GVK9ZHZSMR*czE3NTA4NzA1MDUkbzEkZzAkdDE3NTA4NzA1MDUkajYwJGwwJGgw>`__"
+whitepaper and assumes you have read the whitepaper.
+
+Canton Network tokenomics is based on an *Activity Record* which
+identifies a party that performed an action which provides value to the
+network. Creating an activity record and minting the associated Canton
+Coin (CC) are two distinct steps. The creation and minting steps are
+performed in a cycle that is called a *round* which has five phases. In
+the first phase, any fee values for that round are written to the ledger
+(the fees can be obtained from the `Scan State
+API <https://docs.sync.global/app_dev/scan_api/scan_current_state_api.html>`__).
+The second phase is called the *activity summary* and it is when
+activity records are created. The next phase calculates a *minting
```

Comment:
This is wrong. In the second phase activity records are summed up to compute the following summary values: https://github.com/hyperledger-labs/splice/blob/332e06a7ae9e13fde5bba0bf7dcb059aa36f979e/daml/splice-amulet/daml/Splice/Issuance.daml#L53-L60 

For featured app markers the recording happens in two steps: marker is created as part of tx, and then converted by the SVs to an activity record. Explained in https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0047/cip-0047.md#overview